src/transformers/models/vilt/processing_vilt.py

```diff
@@ -0,0 +1,175 @@
+            raise ValueError(
+                f"`tokenizer` has to be of type {BertTokenizer.__class__} or {BertTokenizerFast.__class__}, but is {type(tokenizer)}"
+            )
+
+        self.feature_extractor = feature_extractor
+        self.tokenizer = tokenizer
+        self.current_processor = self.feature_extractor
+
+    def save_pretrained(self, save_directory):
+        """
+        Save a ViLT feature_extractor object and BERT tokenizer object to the directory `save_directory`, so that it
+        can be re-loaded using the [`~ViltProcessor.from_pretrained`] class method.
+
+        <Tip>
+
+        This class method is simply calling [`~feature_extraction_utils.FeatureExtractionMixin.save_pretrained`] and
+        [`~tokenization_utils_base.PreTrainedTokenizer.save_pretrained`]. Please refer to the docstrings of the methods
+        above for more information.
+
+        </Tip>
+
+        Args:
+            save_directory (`str` or `os.PathLike`):
+                Directory where the feature extractor JSON file and the tokenizer files will be saved (directory will
+                be created if it does not exist).
+        """
+
+        self.feature_extractor.save_pretrained(save_directory)
+        self.tokenizer.save_pretrained(save_directory)
+
+    @classmethod
+    def from_pretrained(cls, pretrained_model_name_or_path, **kwargs):
+        r"""
+        Instantiate a [`ViltProcessor`] from a pretrained ViLT processor.
+
+        <Tip>
+
+        This class method is simply calling ViltFeatureExtractor's
+        [`~feature_extraction_utils.FeatureExtractionMixin.from_pretrained`] and BertTokenizer's
+        [`~tokenization_utils_base.PreTrainedTokenizer.from_pretrained`]. Please refer to the docstrings of the methods
+        above for more information.
+
+        </Tip>
+
+        Args:
+            pretrained_model_name_or_path (`str` or `os.PathLike`):
+                This can be either:
+
+                - a string, the *model id* of a pretrained feature_extractor hosted inside a model repo on
+                  huggingface.co. Valid model ids can be located at the root-level, like `bert-base-uncased`, or
+                  namespaced under a user or organization name, like `dbmdz/bert-base-german-cased`.
+                - a path to a *directory* containing a feature extractor file saved using the
+                  [`~SequenceFeatureExtractor.save_pretrained`] method, e.g., `./my_model_directory/`.
+                - a path or url to a saved feature extractor JSON *file*, e.g.,
+                  `./my_model_directory/preprocessor_config.json`.
+            **kwargs
+                Additional keyword arguments passed along to both [`SequenceFeatureExtractor`] and
+                [`PreTrainedTokenizer`]
+        """
+        feature_extractor = ViltFeatureExtractor.from_pretrained(pretrained_model_name_or_path, **kwargs)
+        tokenizer = BertTokenizer.from_pretrained(pretrained_model_name_or_path, **kwargs)
```

Comment:
@sgugger as you can see here, I'm instantiating a slow tokenizer from the hub.

However, it may be that people want to use the fast tokenizer instead. Should I add an argument `use_fast` to the `from_pretrained` method here?

Reply:
Why use the slow class when we have a fast one available? IMO there should be no arg, just `BertTokenizerFast`.

Reply:
Ok, removed `BertTokenizer` in favor of `BertTokenizerFast`.